x/upgrade/client/cli/query.go

```diff
@@ -110,3 +112,45 @@ func GetAppliedPlanCmd() *cobra.Command {
 
 	return cmd
 }
+
+// GetVersionMapCmd returns the version map from state
+func GetVersionMapCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "versionmap [optional module_name]",
+		Short: "get module version map",
+		Long: "Gets a map of module names and their respective consensus versions.\n" +
+			"Following versionmap with a specific module name will return only the\n" +
+			"version of that module.",
+		Args: cobra.MinimumNArgs(0),
```

Comment:
Would also add a max N args of 1

Reply:
i'll just replace it with maxnargs(1), im now realizing the min(0) does nothing lol